### PR TITLE
Changed README to reference the new documentation

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,155 +31,12 @@ branch of the plugin can be obtained by updating the Gemfile in this way:
...
-On save, a reboot is sent to the discovered host, after which it should reboot into
-the installer for the chosen OS, and finally into the installed OS.
-
-Delete a machine and reboot it to have it move back to the Discovery Pool.
-
 # Permissions
 
 The plugin will create a Role called `Discovery` when first started. This can
 be assigned to roles for non-admins to allow them to use the discovery plugin.
 Alternatively assign the `:perform_discovery` permission to an existing Role.
```
Comment (lzap) [inline README.md]: Mind moving this into the main docs? Let's have everything there.

**Comment by lzap** [comment]
You know what, I will do the permissions. Just delete it from here.

**Comment by orrabin** [comment]
@lzap I removed permissions

**Comment by lzap** [comment]
Please remove the compatibility table as well, moving to the docs too.

**Comment by orrabin** [comment]
@lzap how about `Latest code`?

**Comment by lzap** [comment]
We can leave that... thanks. Will merge today.

**Comment by lzap** [comment]
Pushed as 9122bea thank you!

**Pull request merged**